### ISSUE TITLE
Firefox tab container breaks message decrypt

### ISSUE THREAD

**Issue by shenada** [FlowCrypt/flowcrypt-browser#3838]
### Description:
Firefox tab container breaks message decrypt but works with new tab open as 'No Container'.

### Test Environment:
Arch x86_64
Windows 10 / Linux
Firefox 87-90

### How to reproduce issue:
1. install Mozilla official extension [Firefox Multi-Account Containers](https://addons.mozilla.org/en-US/firefox/addon/multi-account-containers/).
2. install [FlowCrypt Firefox extension](https://flowcrypt.com/download).
3. Login to Gmail in any tab container then 'Secure Compose' a new test encrypted mail to self.
4. open the test mail

### Expected result:
Decrypted mail message shows.

### Actual result:
'Could not load message due to network error. retry' shows and 'retry' never works.

**Comment by tomholub** [comment]
Thank you for the report - indeed the extension is not compatible with Firefox Containers or Google Tab Groups at the moment.

**Comment by tomholub** [comment]
We'd like to eventually make it compatible, but it's not on current roadmap so I'm unable to say if/when this could be done. One workaround would be to use a separate Firefox profile that does not use containers, to run FlowCrypt. That will automatically "containerize" it from other usage without affecting function.

**Comment by Tanman897** [comment]
@shenada yeah, I had told the developers about this too like I told you on the Google Containers github page. I think that they really don't care about this issue. It has gone unfixed for the better part of a year now and seems like it would be a priority considering that most PGP users would probably be using Firefox and Containers. It is a real shame as it essentially breaks 50% of the functionality of this PGP solution for me.

@tomholub with all due respect for you trying to at least propose an alternate solution, having a whole separate Firefox account just to work around this issue is beyond practical since Firefox is not designed for multiple accounts to be logged in at once and logging in/out of different Firefox accounts EVERY time I want to check Gmail is a total pain.

**Comment by tomholub** [comment]
I understand the inconvenience and sympathize with it. I'll be happy to review merge a PR contribution that fixes it. Please understand that we are providing a software solution for free, and have limited resources to address user requests, especially the more complicated ones coming from a minority of users on a minority platform.
